packages/block-library/src/latest-posts/edit.js

```diff
@@ -209,23 +226,28 @@ class LatestPostsEdit extends Component {
 					<QueryControls
 						{ ...{ order, orderBy } }
 						numberOfItems={ postsToShow }
-						categoriesList={ categoriesList }
-						selectedCategoryId={ categories }
 						onOrderChange={ ( value ) =>
 							setAttributes( { order: value } )
 						}
 						onOrderByChange={ ( value ) =>
 							setAttributes( { orderBy: value } )
 						}
-						onCategoryChange={ ( value ) =>
-							setAttributes( {
-								categories: '' !== value ? value : undefined,
-							} )
-						}
 						onNumberOfItemsChange={ ( value ) =>
 							setAttributes( { postsToShow: value } )
 						}
 					/>
+					<FormTokenField
```

Comment:
We should check if the website has _any_ categories before displaying this field:

```JSX
{ categoriesList.length > 0 && (
                                            <FormTokenField ...
```